### PR TITLE
adding command to run the litterate file

### PR DESCRIPTION
**Pull request by nrolland**

### CODE REVIEW

File: doc/tutorial/Server.lhs [hunk 1]
```diff
@@ -149,7 +149,7 @@ repo](http://github.com/haskell-servant/servant) and look at the
 *doc/tutorial* directory. This code (the source of this web page) is in
 *doc/tutorial/Server.lhs*.
 
-If you run it, you can go to `http://localhost:8081/users` in your browser or
+If you run it (using `ghci -pgmL markdown-unlit` or `stack exec -- ghci -pgmL markdown-unlit` and then evaluating `main` into the repl), you can go to `http://localhost:8081/users` in your browser or
```
Comment (phadej) [inline doc/tutorial/Server.lhs]: How to run with `cabal new-run` ?

Comment (nrolland) [reply doc/tutorial/Server.lhs]: what would be the complete command ?

Comment (nrolland) [reply doc/tutorial/Server.lhs]: maybe some instruction in the `examples.md` above and/or in the `lhs` files could be useful to run the literate files.

I doubt newcomers would come up with the idea of running `stack exec -- ghci -pgmL markdown-unlit`  by themselves..

Comment (phadej) [reply doc/tutorial/Server.lhs]: I'd change `tutorial.cabal` so there would be a `executable tutorial-server`, so people can just `cabal new-run tutorial-server`.

Comment (nrolland) [reply doc/tutorial/Server.lhs]: as of today, after a binary dependencies installation (say `hsc2hs`), `cabal new-repl doc/tutorial/` seem to load the tutorial correctly

**Comment by nrolland** [comment]
I missed the index.rst file, and was under the assumption that the tutorial should be launched in the stack.yaml environment from two level up. So I'll just close this !

**Pull request closed without merging**